src/wampy.js

```diff
@@ -2061,51 +2061,44 @@ class Wampy {
     /**
      * RPC invocation cancelling
      *
-     * @param {int} reqId RPC call request ID
+     * @param {int} requestId RPC call request ID
      * @param {object} [advancedOptions] - optional parameter. Must include any or all of the options:
      *                          { mode: string|one of the possible modes:
      *                                  "skip" | "kill" | "killnowait". Skip is default.
      *                          }
      *
      * @returns {Boolean}
      */
-    cancel (reqId, advancedOptions) {
-        const options = {};
-
-        if (!this._preReqChecks(null, 'dealer')) {
+    cancel (requestId, advancedOptions) {
```

Comment:
Hmm... Well, I don't mind about this renaming. But you changed it only in one function. But there are still 60+ `reqId`s that exists across sources. Also `reqId` is a part of the public status object. I'm for consistency.

Reply:
Ok I will change it back then